lib/daffy_lib/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module DaffyLib
-  VERSION = "0.1.7"
+  VERSION = "0.1.8"
```

Comment:
I considered bumping this to 0.2.0 given this is a non trivial change but not sure if we are following (or want to follow) semver

Reply:
what would you suggest? we don't seem to keep a changelog and I don't think we want to branch off master which is how I would think this would normally be done.